pyPhin.py

```diff
@@ -238,6 +238,17 @@ def getWaterData(self, authToken, deviceUUID, vesselUrl):
 			except:
 				self.logger.error("Not able to access %s with %s",dataType,req.text)
 
+		try:
+			testStrip = False
+			for action in reqJson["vessels"][0]["requiredActions"]:
+				if action["buttonDetails"]["title"] == "Dip a test strip":
+					testStrip = True
+			if testStrip:
+				data["pool"]["test_strip_required"] = True
+			else:
+				data["pool"]["test_strip_required"] = False
+		except:
```

Comment:
you can simplify this statement by saying:

data["pool"]["test_strip_required"] = testStrip

or you can use it without the temp value of "testStrip"